client/src/App.css

```diff
@@ -1,42 +1,137 @@
-/* #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+body {
+  color: rgb(44, 41, 54);
+  font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+    "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+.App-title {
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  color: rgb(57, 54, 70);
+  letter-spacing: 3px;
+  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
+  font-size: 3rem;
+  text-transform: uppercase;
+}
+h2 {
+  margin: 0;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+h3 {
+  border: 2px solid #333;
+  border-radius: 10px;
+  padding: 10px;
+  background-color: #f0f0f0;
+  color: #333;
+}
+
+p {
+  font-size: 16px;
+  color: #666;
+  margin-bottom: 10px;
 }
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+
+/* Radio Buttons */
+.radio-label {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  margin-right: 10px;
+  margin-bottom: 10px;
+}
+
+.radio-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.emo-radio {
+  cursor: pointer;
+  width: 24px;
+  height: 24px;
+}
+
+/* Hide the default radio button visually, but keep it for accessibility */
+.emo-radio input[type="radio"] {
+  position: absolute;
+  opacity: 0;
+}
+
+/* Apply styles to the list items */
+ul {
+  list-style: none;
+  padding: 0;
+  display: flex;
 }
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+li {
+  margin-bottom: 20px;
+  margin-right: 20px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
+.track-info {
+  margin-left: 10px;
 }
 
-.card {
-  padding: 2em;
+/* Apply styles to the links */
+a {
+  text-decoration: none;
+  color: #007bff;
 }
 
-.read-the-docs {
-  color: #888;
-} */
+a:hover {
+  text-decoration: underline;
+}
+
+/* decision btn styles */
+.let-go-btn,
+.embrace-btn {
+  display: inline-block;
+  padding: 35px 70px;
+  font-size: 25px;
+  font-weight: bold;
+  text-align: center;
+  text-decoration: none;
+  cursor: pointer;
+  border: 2px solid rgb(63, 46, 62);
+  border-radius: 5px;
+  color: rgb(248, 247, 246);
+  background-color: #1f1f1f;
+  transition: border-color 0.3s, color 0.3s, background-color 0.3s,
+    transform 0.3s;
+  transform-origin: center;
+}
+
+.let-go-btn:hover,
+.embrace-btn:hover {
+  transform: scale(1.1);
+}
+
+.submit-btn {
+  display: inline-block;
```

Comment:
buttons are by default `inline-block`. You could try removing this. Although I'm curious if that changes the width span of the button.

Reply:
Removed and it did not change anything! Thanks

Reply:
[8ee7101](https://github.com/destineevaldivia/EmoHelper/pull/3/commits/8ee71014fd9ad13b71c8bf444b032c557a3ddd75)